### ISSUE TITLE
Recurring issues

### ISSUE THREAD

**Issue by test2a** [tillywork/tillywork#28]
Would it be possible to have recurring issues 

Like I want to have 10 issues again on 2nd of every month in to do already assigned to persons

**Comment by topherjaynes** [comment]
Is this sort of like a calendar event setup? I'd like this to repeat (weeks, months, or days of the month). Are there any good examples? For example, would paying the electrical bill on the second Tuesday of each month be a good example?

**Comment by test2a** [comment]
@topherjaynes yes. that and this would land in the to do of the family member assigned to pay the bills.

other examples would be regular office meetings with changing name depending on the current month for example.

in the case of electricity, the task would be next month called "electric bill for january" assigned to bob. next month "electric bill for february" assigned to bob.

https://www.projectmanager.com/blog/managing-recurring-tasks
here is how these people do recurring tasks
